package/MDAnalysis/analysis/align.py

```diff
@@ -293,15 +303,34 @@ def _fit_to(mobile_coordinates, ref_coordinates, mobile_atoms,
         array of xyz coordinate of mobile center of mass
     ref_com: ndarray
         array of xyz coordinate of reference center of mass
-    weights : numpy array (optional)
+    weights : ndarray (optional)
         Array to be used for weighted rmsd
```

Comment:
Ok, but this one cannot use "mass" because `rotation_matrix()` is pretty bare-bones and just uses whatever array is passed through. I'd keep it that way for the lower level functions.